tools/wptserve/tests/functional/test_server.py

```diff
@@ -40,13 +42,28 @@ def handler(request, response):
 
         self.assertEqual(cm.exception.code, 500)
 
+    def test_many_headers(self):
+        @wptserve.handlers.handler
+        def handler(request, response):
+            assert len(request.headers) == 260
```

Comment:
Can you document why its 260 here but 256 below? I assume that `request` adds some headers? (Could that change in the future without us knowing, i.e. could this be a source of brittleness?)

Reply:
Tweaked the test to be more reliable.